Sources/StreamChat/Database/DTOs/UserDTO_Tests.swift

```diff
@@ -373,4 +373,82 @@ class UserDTO_Tests: XCTestCase {
             NSSortDescriptor(key: "userRoleRaw", ascending: true)
         )
     }
+
+    func test_DTO_triggerMembersUpdate() throws {
```

Comment:
The name of the method should reflect what you test. Maybe something like `test_changingUserDTO_triggersMemberDTOUpdate`?